ts/src/profiler.ts

```diff
@@ -72,6 +75,29 @@ export interface RequestProfile {
   labels?: {instance?: string};
 }
 
+/**
+ * @return number indicated by backoff if the response indicates a backoff and
+ * that backoff is greater than 0. Otherwise returns undefined.
+ */
+// tslint:disable-next-line: no-any
+function getServerResponseBackoff(response: any): number|undefined {
```

Comment:
I did this because http.ServerResponse doesn't have a field  body. So if I use http.ServerResponse as the type, then I would have to cast the response to any throughout this function.
Would that be preferable?

Reply:
I've switched to using http.ServerResponse as the type.